variation-commons-core/src/main/java/uk/ac/ebi/eva/commons/core/models/VariantClassifier.java

```diff
@@ -33,20 +35,28 @@
      * @return Type of the variant
      */
     public static VariantType getVariantClassification(String reference, String alternate, int subSNPClass) {
-        String alphaRegEx = "[A-Z]+";
+        Pattern alphaRegExPattern = Pattern.compile("[A-Z]+");
```

Comment:
Maybe the Java compiler optimizes this, but this should be put at the class level. It can be a static field, or you can make the class concrete and this method non-static, up to you.